<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitProjectWidget</class>
 <widget class="QWidget" name="GitProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projets Git</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="repositoriesGroup">
     <property name="title">
      <string>Dépôts configurés</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="repoToolbarLayout">
        <item>
         <widget class="QComboBox" name="repoComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addRepoButton">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset theme="list-add"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editRepoButton">
          <property name="text">
           <string>Éditer</string>
          </property>
          <property name="icon">
           <iconset theme="document-edit"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeRepoButton">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="syncButton">
          <property name="text">
           <string>Synchroniser</string>
          </property>
          <property name="icon">
           <iconset theme="view-refresh"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="repoInfoLabel">
        <property name="text">
         <string>Aucun dépôt configuré</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="issuesGroup">
     <property name="title">
      <string>Issues</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="filterLayout">
        <item>
         <widget class="QCheckBox" name="showOpenIssues">
          <property name="text">
           <string>Ouvertes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showClosedIssues">
          <property name="text">
           <string>Fermées</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="placeholderText">
           <string>Rechercher...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="issuesListWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="issueActionsLayout">
        <item>
         <widget class="QPushButton" name="createIssueButton">
          <property name="text">
           <string>Créer une issue</string>
          </property>
          <property name="icon">
           <iconset theme="document-new"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createTaskFromIssueButton">
          <property name="text">
           <string>Créer tâche liée</string>
          </property>
          <property name="icon">
           <iconset theme="go-jump"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
